client/src/core/format.js

```diff
@@ -129,7 +129,7 @@ const percentage = (precision, val, lang, options) => {
 };
 
 // 'smart' percent formatter that scales decimal places (up to 3) based on the value
-const percentage_smart = (min_precision, val, lang, options) => {
+const smart_percentage = (min_precision, val, lang, options) => {
   const smart_precision = val < 0.001 ? 3 :
     (val < 0.01 ? 2 : 1);
```

Comment:
bracket consistency might help with clarity (just adding brackets to the top statement)